src/controllers/apiController.ts

```diff
@@ -341,11 +341,11 @@ export class ApiController extends BaseController {
    *
    * @returns BXML string
    */
-  static generateTransferBxml(deviceToken: string, sipUri='sip:sipx.webrtc.bandwidth.com:5060'): string {
-    return '<?xml version="1.0" encoding="UTF-8" ?>\n'
-        + '<Response>'
-        + this.generateTransferBxmlVerb(deviceToken, sipUri)
-        + '</Response>';
+  static generateTransferBxml(deviceToken: string, sipUri='sip:sipx.webrtc.bandwidth.com:5060', voiceCallId: string): string {
```

Comment:
Something we should talk about as a group is whether to make this parameter optional. I lean towards no. However, sipUri is an optional parameter so I think we'd have to have voiceCallId come before sipUri, since all required parameters are supposed to be listed before optional ones.

Reply:
After reflecting, I agree that the voiceCallId should be mandatory if someone upgrades to the new SDK. If there was some reason they couldn't supply the voiceCallId, they could supply their own Bxml.